pylint/checkers/variables.py

```diff
@@ -1203,35 +1203,43 @@ def leave_listcomp(self, _):
         # do not check for not used locals here
         self._to_consume.pop()
 
+    def leave_functiondef(self, node):
+        self._check_metaclasses(node)
+        super(VariablesChecker3k, self).leave_functiondef(node)
+
     def leave_module(self, node):
-        """ Update consumption analysis variable
-        for metaclasses.
-        """
-        module_locals = self._to_consume[0][0]
-        module_imports = self._to_consume[0][1]
-        consumed = {}
+        self._check_metaclasses(node)
+        super(VariablesChecker3k, self).leave_module(node)
+
+    def _check_metaclasses(self, node):
```

Comment:
I feel like naming of variables  `consumed `, `scope_not_consumed_locals`, `scope_locals_dict` should be better.